packages/grid/x-data-grid-pro/src/utils/releaseInfo.ts

```diff
@@ -0,0 +1,15 @@
+import { ponyfillGlobal } from '@mui/utils';
+
+export const getReleaseInfo = () => {
```

Comment:
This utility function is not very usefull for the grid because there is currently only one call
But for the pickers, we have several commercial components so it's better to have a centralized place.
And keeping things similar in all packages is cleaner I think.

This code can't be in `x-license-pro` because the `string-replace-loader` build step must be done with the date of the build of each package.